test/functional/legacy/001_script_test_spec.lua

```diff
@@ -0,0 +1,33 @@
+-- First a simple test to check if the test script works.
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+local eval = helpers.eval
+
+describe('test script', function()
+  setup(clear)
+
+  it('is working', function()
+    -- Write a single line to test.out to check if testing works at all.
+    execute('%d')
+    feed('athis is a test<esc>:w! test.out<cr>')
```

Comment:
AFAIK writing `test.out` is not necessary in the migrated functional tests. Is test001 a special case where it's needed?

compare https://github.com/neovim/neovim/pull/1462/files